src/cljs/harja/ui/viesti.cljs

```diff
@@ -95,8 +95,15 @@
           (when (or
                   (= :varoitus luokka)
                   (= :neutraali-ikoni-keskella luokka))
-            [:span {:style {:padding-left "16px"}}
-             [ikonit/sulje]])]])
+            [:span {:style {:padding-left "10px"}}
+             [:button {:style {:color "black"
+                               :opacity 1
+                               :cursor "pointer"
+                               :background "transparent"
+                               :border 0}
+                       :type "button"
+                       :id "btn-sulje"}
```

Comment:
Nämä  voisi heittää jo .less tiedostoon.
Etsi koodipesästä esim missä  yllä käytetty .toast-viesti-container on definattu (näyttää olevan styles.less tiedosto)

On OK näinkin, mutta minusta siistimpää jos näitä tyylittelyjä heitettäisiin aina Lessiin.